deck/forms.py

```diff
@@ -26,9 +26,12 @@ class Meta:
         widgets = {
             'title': forms.TextInput(attrs={'class': 'inline-input'}),
             'due_date': CustomDateTimeWidget(attrs={
-                'id': 'id_due_date',
                 'class': 'inline-input',
-                'placeholder': 'Due Date'
+                'placeholder': 'Due Date',
```

Comment:
This should use the translation function `uggetext`, something like: `_('Due Date')`